cli/api.js

```diff
@@ -90,40 +90,32 @@ function getPackageName(plugin) {
 
 function install(plugin, locally) {
   const array = locally ? getLocalPlugins() : getPlugins();
-  return new Promise((resolve, reject) => {
-    existsOnNpm(plugin)
-      .then(() => {
-        if (isInstalled(plugin, locally)) {
-          return reject(`${plugin} is already installed`);
-        }
-
-        array.push(recast.types.builders.literal(plugin));
-        save()
-          .then(resolve)
-          .catch(err => reject(err));
-      })
-      .catch(err => {
-        const {statusCode} = err;
-        if (statusCode && (statusCode === 404 || statusCode === 200)) {
-          return reject(`${plugin} not found on npm`);
-        }
-        return reject(`${err.message}\nPlugin check failed. Check your internet connection or retry later.`);
-      });
-  });
+  return existsOnNpm(plugin)
+    .then(() => {
+      if (isInstalled(plugin, locally)) {
+        return Promise.reject(`${plugin} is already installed`);
```

Comment:
Is't this the same as ```throw new Error(`${plugin} is already installed`)```

Reply:
No, using [`Promise.reject`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/reject) returns a rejected promise, but we could do
```js
return Promise.reject(new Error(`${plugin} is already installed`));
```
instead.